sentences/de/_common.yaml

```
@@ -229,6 +237,7 @@ expansion_rules:
   artikel: "[<artikel_bestimmt>|<artikel_unbestimmt>]"
   name: "<artikel> {name}"
   area: "[in|an|auf|bei] [<artikel>|im|am|<von_dem>] {area}[s]"
+  floor: "[in|auf] [<artikel>|im|<von_dem>] [Stock[werk]|Etage|Geschoss] {floor}"
```

i'm not sure, if we really need `[Stock[werk]|Etage|Geschoss]` in here (_you can see it on the weird sounding test sentences_ 🙈 ) - we may just keep `[Geschoss]` as suffix to allow a floor called `Keller` be also targeted by "Keller Geschoss" 🤔 

```suggestion
  floor: "[in|auf] [<artikel>|im|<von_dem>] {floor} [Geschoss]"
```

---

What if someone calls his floors just by numbers - like "2"? 😄 Then it would be useful to have it as a prefix but I don't insist on having it 😃

---

I was also thinking about just numbered floor names ... in English it is easy with `{floor} [floor]` but thanks to the German complex grammar we need to have something like `[Stock[werk]|Etage|Geschoss] {floor}[. Stock[werk]|Etage|Geschoss]` 🙈